torch/_subclasses/fake_tensor.py

```diff
@@ -85,6 +85,11 @@ class DataDependentOutputException(RuntimeError):
     aten._resize_output.out,
 )
 
+_nested_constructors = (
```

Comment:
Unrelated, it would be nice if you could tell in `_nested_tensor_from_tensor_list` native_function schema list that the output was a NestedTensor..

Reply:
I agree and can add a comment for now. Eventually perhaps it makes sense to have `NestedTensor` be a supported type within `native_functions.yaml`?